src/plugin/protocol-smb/src/test/org/apache/nutch/protocol/smb/TestSmbProtocol.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package plugin.protocol-smb.src.test.org.apache.nutch.protocol.smb;
+
+// Hadoop imports
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.io.Text;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TestSmbProtocol {
+    protected static final Logger LOG = LoggerFactory.getLogger(TestSmbProtocol.class);
+    
+    @Before
+    public void setUp() {
+        LOG.warn("setUp()");
+        Assert.fail();
```

Comment:
These tests cannot be committed in the failing state. Thy will destabilize the CI builds. 
Previously I suggested using [test containers](https://issues.apache.org/jira/browse/NUTCH-2856?focusedCommentId=17467111&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17467111). 
If docker is not available on the host machine when tests are being run then we could use the [@Testcontainers(disabledWithoutDocker = true)](https://java.testcontainers.org/quickstart/junit_5_quickstart/#4-additional-attributes) syntax.